node_cli/mirage/mirage_node.py

```diff
@@ -22,16 +22,21 @@
 import time
 
 from node_cli.configs import SKALE_DIR, RESTORE_SLEEP_TIMEOUT
-from node_cli.core.host import save_env_params
+from node_cli.configs.env import SKALE_DIR_ENV_FILEPATH
 from node_cli.core.node import compose_node_env
-from node_cli.utils.decorators import check_not_inited
+from node_cli.core.host import save_env_params
+from node_cli.core.static_config import get_static_params
+from node_cli.mirage.record.chain_record import ChainRecord
+from node_cli.operations import restore_mirage_op
+from node_cli.utils.decorators import check_inited, check_not_inited
 from node_cli.utils.exit_codes import CLIExitCodes
 from node_cli.utils.helper import error_exit
 from node_cli.utils.node_type import NodeType
-from node_cli.operations import restore_mirage_op
+from node_cli.utils.texts import safe_load_texts
 
 
 logger = logging.getLogger(__name__)
+TEXTS = safe_load_texts()
```

Comment:
mb let's add a proper return type to `safe_load_texts` so mypy check will not crash when accessing `TEXTS`?

```
Object of type "None" is not subscriptablePylance[reportOptionalSubscript](https://github.com/microsoft/pylance-release/blob/main/docs/diagnostics/reportOptionalSubscript.md)
```

Reply:
addressed